### PR TITLE
Move to EmerisHQ

### PR DESCRIPTION
**Pull request by pitasi**

### CODE REVIEW

**Review by lukitsbrian — APPROVED** [review]

**Comment by pitasi** [comment]
I based this on top of #73 otherwise I cannot build `api-server` (emeris-utils changed name *after* breaking changes of k8s config, so in order to import emeris-utils using the new path I also have to use the new k8s api).

File: Dockerfile [hunk 1]
```diff
@@ -2,7 +2,7 @@ FROM golang:1.17 as builder
 
 ARG GIT_TOKEN
 
-RUN go env -w GOPRIVATE=github.com/allinbits/*
+RUN go env -w GOPRIVATE=github.com/emerishq/*
```
Comment (DeshErBojhaa) [inline Dockerfile]: Doesn't docker build require `allinbits/starport-operator`?

Comment (pitasi) [reply Dockerfile]: yes, thanks for spotting this, fixed

**Review by DeshErBojhaa — APPROVED** [review]
LGTM!

**Review by akhilkumarpilli — APPROVED** [review]

**Pull request merged**